ext/web/06_streams.js

```diff
@@ -193,6 +207,20 @@
     return isFakeDetached in O;
   }
 
+  /**
+   * @param {ArrayBufferLike} O
+   * @returns {boolean}
+   */
+  function canTransferArrayBuffer(O) {
+    assert(typeof O === "object");
+    assert(O instanceof ArrayBuffer || O instanceof SharedArrayBuffer);
```

Comment:
`SharedArrayBuffer`s aren't transferable, and I don't think this spec algorithm is meant to be passed a SAB.